test/src/loaders/workspace_uploader/workspace_uploader_test.py

```diff
@@ -587,3 +590,89 @@ def test_query_workspace_with_load_id_mass(setup_and_teardown):
             "includeMetadata": 1
         }
     )
+
+
+def test_query_workspace_with_load_id_mass_genome(setup_and_teardown):
+    # Test case 1: Valid scenario - test with genome objects and assembly_objs_only is set to False
+    ws = create_autospec(Workspace, spec_set=True, instance=True)
+    load_id = "998"
+    assembly_objs_response = {
+        "infos": [
+            [1, 'assembly_1', 'KBaseGenomeAnnotations.Assembly-6', 'time', 75, 'user', 42, 'wsname', 'md5', 78, {'foo': 'bar'}],
+            [2, 'assembly_2', 'KBaseGenomeAnnotations.Assembly-10', 'time', 75, 'user', 42, 'wsname', 'md5', 78, {}]
+        ]
+    }
+
+    genome_objs_response = {
+        "infos": [
+            [1, 'genome_1', 'KBaseGenomes.Genome-6', 'time', 75, 'user', 42, 'wsname', 'md5', 78,
+             {"load_id": load_id, "Assembly Object": "1/1/1"}],
+            [4, 'genome_2', 'KBaseGenomes.Genome-9.3', 'time', 75, 'user', 42, 'wsname', 'md5', 78,
+             {"load_id": load_id, "Assembly Object": "2/2"}]
```

Comment:
The assembly object ref is always pulled from the object and is annotated as a WS id so it will always be fully qualified

Reply:
hmmm can you clarity more?

Reply:
It'll never be `2/2` since the workspace rewrites any `@id ws` annotated fields to full UPAs, and the `assembly_ref` field, which is where the `Assembly Object` meta value comes from, is an `@id ws` field

Reply:
👍 fixed.

Reply:
:+1: